owncloudData/src/test/java/com/owncloud/android/data/spaces/repository/OCSpacesRepositoryTest.kt

```diff
@@ -350,6 +351,31 @@ class OCSpacesRepositoryTest {
         }
     }
 
+    @Test
+    fun `editSpaceImage updates the space image correctly`() {
+        every {
+            remoteSpacesDataSource.editSpaceImage(
+                accountName = OC_ACCOUNT_NAME,
+                spaceId = OC_SPACE_PROJECT_WITH_IMAGE.id,
+                imageId = OC_SPACE_SPECIAL_IMAGE.id
+            )
+        } returns OC_SPACE_PROJECT_WITH_IMAGE
+
+        ocSpacesRepository.editSpaceImage(
+            accountName = OC_ACCOUNT_NAME,
+            spaceId = OC_SPACE_PROJECT_WITH_IMAGE.id,
+            imageId = OC_SPACE_SPECIAL_IMAGE.id
+        )
+
+        verify(exactly = 1) {
+            remoteSpacesDataSource.editSpaceImage(
+                accountName = OC_ACCOUNT_NAME,
+                spaceId = OC_SPACE_PROJECT_WITH_IMAGE.id,
+                imageId = OC_SPACE_SPECIAL_IMAGE.id
+            )
+        }
+    }
+
```

Comment:
Tests added are OK, just as a question: would it make sense to add negative tests? i mean, to check the correct response if the action failed (image editing returned an error, a null or so)